pkg/scheduler/core/generic_scheduler.go

```diff
@@ -903,11 +904,44 @@ func pickOneNodeForPreemption(nodesToVictims map[*v1.Node]*schedulerapi.Victims)
 			lenNodes2++
 		}
 	}
+	if lenNodes2 == 1 {
+		return minNodes2[0]
+	}
+
+	// There are a few nodes with same number of pods.
+	// Find the node that satisfies latest(earliestStartTime(all highest-priority pods on node))
+	earliestStartTime := util.GetEarliestPodStartTime(nodesToVictims[minNodes2[0]])
```

Comment:
sorry, I suggested a bad name in my previous review for this variable. This is actually the `lastestStartTime`, not the earliest. Could you please rename it to "latestStartTime"?